### PR TITLE
Mutations for sum-7!

### PR DESCRIPTION
**Pull request by jems41**
comments/suggestions much appreciated! for assignment 4 SF-1

### CODE REVIEW

File: sum_7/solution.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+def sum_7(list_a, list_b):
+    count = 0
+    # assigns a tuple that takes in all the combinations
+    pairs = [(int_a, int_b) for int_a in list_a for int_b in list_b]
+
+    #directly additions and test if the pair is a sum of 7
+    for i in pairs
```
Comment (khanz00024) [inline sum_7/solution.py:7]: you forgot to put : after for i in pairs......its a huge mistake

File: sum_7/mutation_1.py
Comment (khanz00024) [inline sum_7/mutation_1.py:1]: it works

File: sum_7/mutation_2.py
Comment (khanz00024) [inline sum_7/mutation_2.py:1]: it works too but arent we supposed to always return....nvm idk

File: sum_7/mutation_3.py
Comment (khanz00024) [inline sum_7/mutation_3.py:1]: it works

File: sum_7/solution.py (hunk 1 above)
Comment (khanz00024) [reply sum_7/solution.py:7]: just put the colon :

File: sum_7/mutation_4.py
Comment (khanz00024) [inline sum_7/mutation_4.py:1]: it works

File: sum_7/mutation_5.py
Comment (khanz00024) [inline sum_7/mutation_5.py:1]: it works

**Comment by jems41** [comment]
done!

**Pull request merged**